cfpb/consumerfinance.gov
# P4C: small text fixes for grad path tool

Updates to text in grad path tool.

## Changes
- Add `including the coming school year` to program length question and fix conditional display of text based on answer to progress question
- Move bullet points into notification on costs page
- Fix em dashes and typos


## How to test this PR

1. Check out branch and run gulp
2. On first screen of grad path tool, verify that text of program length question updates correctly based on selection in program progress question (`how long` if first or second option is selected in progress question, `how much longer` if any other option is selected)
3. On costs screen, verify that two bullet points appear in notification
4. Verify that em dashes appear correctly throughout tool 


## Screenshots

### Program length question for first years
<img width="618" alt="Screen Shot 2021-05-10 at 10 47 41 AM" src="https://user-images.githubusercontent.com/778171/117703271-6ac19800-b17e-11eb-9f95-096f1d93f430.png">

### Program length question for non-first-years
<img width="591" alt="Screen Shot 2021-05-10 at 10 47 49 AM" src="https://user-images.githubusercontent.com/778171/117703324-6dbc8880-b17e-11eb-9f22-6118cf6ac948.png">

### Costs screen notification
<img width="622" alt="Screen Shot 2021-05-10 at 10 48 07 AM" src="https://user-images.githubusercontent.com/778171/117703140-42399e00-b17e-11eb-8bf4-e5d15379fc51.png">






## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
